### PR TITLE
Add Stateful example app to blue-green and expand-contract

### PR DESCRIPTION
**Pull request by erkolson**
- Elasticsearch cluster added to blue-green and expand-contract
- Corresponding updates to READMEs
- Update test/make.sh / Makefile

A forthcoming PR will have additional Elasticsearch specific validation.sh scripts for blue-green and expand-contract

### CODE REVIEW

**Comment by erkolson** [comment]
The validation scripts for blue-green and expand-contract have been updated for the Elasticsearch cluster in the second commit.

File: expand-contract-upgrade/cluster_ops.sh [hunk 1]
```diff
@@ -138,16 +162,60 @@ create_cluster() {
...
+  echo "Loading Shakespeare sample data into Elasticsearch"
+  curl -H "Content-Type: application/x-ndjson" \
+    -X POST \
+    --data-binary @"${REPO_HOME}/data/shakespeare.json" \
+    'http://localhost:9200/shakespeare/doc/_bulk?pretty' > /dev/null
+
+  # If we've made it this far the data is loaded
+  echo "Sample data successfully loaded!"
+
+  pkill -P $$
```
Comment (techgnosis) [inline expand-contract-upgrade/cluster_ops.sh]: Is killing the script going to kill the kubectl in the background?

File: expand-contract-upgrade/validate.sh [hunk 2]
```diff
@@ -133,6 +134,81 @@ validate_nodes() {
...
+  EXPECTED_LINE="Yea, noise? then Ill be brief. O happy dagger!"
+  if ! [[ "${JULIET_LINE}" == "${EXPECTED_LINE}" ]]; then
+    echo "ERROR: Juliet's line is not here!"
+    return 1
+  else
+    echo "Text search succeeded:"
+    echo "Juliet: \"${JULIET_LINE}\""
+  fi
+
+  pkill -P $$
```
Comment (techgnosis) [inline expand-contract-upgrade/validate.sh]: Same thing here. Not sure if killing this script will kill the backgrounded kubectl for the port-forward.

File: expand-contract-upgrade/cluster_ops.sh (hunk 1 above)
Comment (erkolson) [reply expand-contract-upgrade/cluster_ops.sh]: Yes, `-P` searches for processes with a parent pid matching the given number and `$$` is the PID of this executing script.  So, this command will kill anything that was spawned by this script.

File: expand-contract-upgrade/validate.sh (hunk 2 above)
Comment (erkolson) [reply expand-contract-upgrade/validate.sh]: It is not killing `this` script, it kills anything that was spawned/forked from this script.

**Pull request merged**